src/Loader.php

```diff
@@ -885,7 +900,6 @@ public static function add_component_settings( $settings ) {
 		}
 
 		$settings['allowMarketplaceSuggestions'] = WC_Marketplace_Suggestions::allow_suggestions();
-		$settings['connectNonce']                = wp_create_nonce( 'connect' );
```

Comment:
Is this something I missed in a `master` merge?

Reply:
Appears to have been added here https://github.com/woocommerce/woocommerce-admin/commit/3cdeb916efbd6ace710bb1dcb96fff6c69ee5540, and is still in `master`

Reply:
@timmyc Thanks, I restored the line of code.